ReactiveCocoa/Objective-C/UITextView+RACSignalSupport.m

```diff
@@ -35,12 +35,7 @@ - (RACDelegateProxy *)rac_delegateProxy {
 }
 
 - (RACSignal *)rac_textSignal {
-	@weakify(self);
-	RACSignal *signal = [[[[[RACSignal
-		defer:^{
-			@strongify(self);
-			return [RACSignal return:RACTuplePack(self)];
-		}]
```

Comment:
But `return:` retains its value. This changes the memory semantics of `rac_textSignal`.